frontend/src/pages/cohorts/CohortsPage.tsx

```diff
@@ -112,6 +121,8 @@ function cohortFilterWhereVariables(parsedSearchVals: string[]): CohortWhere[] {
       },
     ];
   }
+
+  return [];
```

Comment:
Return an empty array when the user isn't searching anything. Without doing this, we end up sending several empty `where` variable values to the server in each request (see the "Before" content below).

I'm adding this fix to the other views as well (i.e. Requests, Patients, and Samples).

<table>
<tr>
<th>Before</th>
<th>After</th>
</tr>
<tr>
<td>
  
```json
{
  "options": {
    "sort": []
  },
  "hasCohortCompleteCohortCompletesOptions2": {
    "sort": [
      {
        "date": "DESC"
      }
    ]
  },
  "where": {
    "OR": [
      {},
      {
        "hasCohortCompleteCohortCompletes_SOME": {}
      },
      {
        "hasCohortCompleteCohortCompletes_SOME": {}
      },
      {
        "hasCohortCompleteCohortCompletes_SOME": {}
      },
      {
        "hasCohortCompleteCohortCompletes_SOME": {}
      },
      {
        "hasCohortCompleteCohortCompletes_SOME": {}
      },
      {
        "hasCohortCompleteCohortCompletes_SOME": {}
      },
      {
        "hasCohortCompleteCohortCompletes_SOME": {}
      },
      {
        "hasCohortSampleSamples_SOME": {
          "hasMetadataSampleMetadata_SOME": {}
        }
      }
    ]
  },
  "cohortsConnectionWhere2": {
    "OR": [
      {},
      {
        "hasCohortCompleteCohortCompletes_SOME": {}
      },
      {
        "hasCohortCompleteCohortCompletes_SOME": {}
      },
      {
        "hasCohortCompleteCohortCompletes_SOME": {}
      },
      {
        "hasCohortCompleteCohortCompletes_SOME": {}
      },
      {
        "hasCohortCompleteCohortCompletes_SOME": {}
      },
      {
        "hasCohortCompleteCohortCompletes_SOME": {}
      },
      {
        "hasCohortCompleteCohortCompletes_SOME": {}
      },
      {
        "hasCohortSampleSamples_SOME": {
          "hasMetadataSampleMetadata_SOME": {}
        }
      }
    ]
  }
}
```
  
</td>
<td>

```json
{
  "options": {
    "sort": []
  },
  "hasCohortCompleteCohortCompletesOptions2": {
    "sort": [
      {
        "date": "DESC"
      }
    ]
  },
  "where": {
    "OR": []
  },
  "cohortsConnectionWhere2": {
    "OR": []
  }
}

























































```

</td>
</tr>
</table>